### ISSUE TITLE
JSON Schema examples not displayed for referenced schemas

### ISSUE THREAD

**Issue by josdejong** [josdejong/jsoneditor#667]
When adding some `examples` to the JSON schema, they are displayed when hovering over the field name. This seems not to work for nested schema's, loaded via reference. 

@AdamVig am I overlooking something, should this work for referenced schema's?

Also it looks like the fields display the description of the object they belong to (hover over `job.salary` in the following example shows "Job description"), I'm not sure if that is the intention.

This can be reproduced by adding some `examples` fields to the JSON schemas in  `07_json_schema_validation.html`, demo:  https://jsbin.com/nohohixuya/edit?html,output

**Comment by AdamVig** [comment]
@josdejong You're right, this seems to be a bug in `Node._findSchema`. I can't tell if I introduced the bug or if it was already there, but I will try to fix it.

**Comment by josdejong** [comment]
Thanks @AdamVig . Yes it's very well possible that the issue was already there.
